Source/GSTransferState.m

```diff
@@ -287,6 +309,17 @@ - (void) setFileURL: (NSURL*)url
 
 - (NSFileHandle*) fileHandle
 {
+  /* Create temporary file and open a fileHandle for writing. */
+  if (!_fileHandle)
+    {
+      NSFileManager *file = [NSFileManager defaultManager];
+      [file createFileAtPath: [[self fileURL] relativePath] contents: nil attributes: nil];
+
+      _fileHandle = [NSFileHandle fileHandleForWritingToURL: [self fileURL] error: NULL];
+
+      RELEASE(file);
```

Comment:
This does not need to be released, you use an already autoreleased NSFileManager.